chrome/locale/en-US/zotero/reader.ftl

```diff
@@ -51,3 +51,23 @@ pdfReader-findInDocumentInput =
     .title = Find
     .placeholder = { pdfReader-find-in-document }
     .aria-description = To turn a search result into a highlight annotation, press { general-key-control }-{ option-or-alt }-1. To turn a search result into an underline annotation, press { general-key-control }-{ option-or-alt }-2.
+
+pdfReader-import-from-epub =
+    .label = Import Ebook Annotations…
+pdfReader-import-from-epub-prompt-title = Import Ebook Annotations
+pdfReader-import-from-epub-prompt-text =
+    { -app-name } found { $count ->
+        [1] { $count } { $tool } annotation
+        *[other] { $count } { $tool } annotations
+    }, last edited { $lastModifiedRelative }.
+    
+    Any { -app-name } annotations that were previously imported from this ebook will be updated.
+pdfReader-import-from-epub-no-annotations-current-file =
+    This ebook does not appear to contain any KOReader or Calibre annotations.
+    
+    If you have another copy in your KOReader or Calibre library, { -app-name } can import annotations from that file.
```

Comment:
And do we need to say this part? Can we just assume people can figure out where their files are?

Also sort of weird to list the specific other applications in the first sentence.

What about just this?

```
This ebook does not appear to contain any importable annotations.

{ -app-name } can import ebook annotations created in Calibre and KOReader.
```